docs/fundamentals/flow/monitoring-flow.md

```diff
@@ -0,0 +1,140 @@
+(monitoring-flow)=
+# Monitor Flow
+ 
+A Jina {ref}`Flow <flow-cookbook>` exposes several core metrics that allow you to have a deeper look
+at what is happening inside it. Metrics allow you to, for example, monitor the overall performance 
+of your Flow, detect bottlenecks, or alert your team when some component of your Flow is down.
+
+Jina Flows exposes metrics in the [Prometheus format](https://prometheus.io/docs/instrumenting/exposition_formats/). This is a plain text format that is understandable by both humans and machines. These metrics are intended to be scraped by
+[Prometheus](https://prometheus.io/), an industry-standard tool for collecting and monitoring metrics.
+
+To visualize your metrics through a dashboard, we recommend [Grafana](https://grafana.com/)
+
+
+## Using the monitoring in a Flow
+
+A {ref}`Flow <flow-cookbook>` is composed of several Pods, namely the Gateway, the Executors, and potentially a Head (see the {ref}`architecture overview <architecture-overview>` for more details). Each of these Pods is its own microservice. These services expose their own metrics using the [Prometheus client](https://prometheus.io/docs/instrumenting/clientlibs/).
+This means that they are as many metrics endpoints as there are Pods in your Flow. 
+
+Let's give an example to illustrate it :
+
+````{tab} via Python API
+
+This example shows how to start a Flow with monitoring enabled via the Python API:
+
+```python
+from jina import Flow
+
+with Flow(monitoring=True, port_monitoring=9090).add(
+    uses='jinahub://SimpleIndexer', port_monitoring=9091
+) as f:
+    f.block()
+```
+````
+
+````{tab} via YAML
+This example shows how to start a Flow with monitoring enabled via yaml:
+
+In a `flow.yaml` file
+```yaml
+!Flow
+with:
+  monitoring: true
+  port_monitoring: 9090
+executors:
+- uses: jinahub://SimpleIndexer
+  port_monitoring: 9091
+```
+
+```bash
+jina flow --uses flow.yaml
+```
+````
+
+This Flow will create two Pods, one for the Gateway, and one for the SimpleIndexer Executor, therefore it will create two 
+metrics endpoints:
+
+* `http://localhost:9090  ` for the gateway
+* `http://localhost:9091  ` for the SimpleIndexer
+
+````{admonition} Default Monitoring port
+:class: hint
+The default monitoring port is `9090`, if you want to enable the monitoring on both the Gateway and the Executors you need to specify
+the `prometheus_port` for the Executors. 
+````
+
+
+Because each Pod in a Flow exposes its own metrics, the monitoring feature can be used independently on each Pod.
+This means that you are not forced to always monitor every Pods of your Flow. For example, you could be only interested in
```

Comment:
```suggestion
This means that you are not forced to always monitor every Pod of your Flow. For example, you could be only interested in
```

Grammarly should've picked this up :angry:

Reply:
might be because of the capitalize letter